src/build/templates/cssFolders.js

```diff
@@ -3,24 +3,24 @@
 module.exports =
   '.monaco-tree-row .content .sub-content .explorer-item.folder-icon {' +
     'padding-left: 22px;' +
-    'background: url(browser/parts/editor/media/icons/Folder_inverse.svg) 1px 4px no-repeat;' +
+    'background: url(browser/parts/editor/media/icons/Folder_inverse.svg) 1px 3px no-repeat;' +
```

Comment:
It turns out that the official implementation uses 3px instead of 4px so all the icons are lifted 1px. Then, as we cannot change this (we're official in this part of the extension) I thought of changing all the icons and lower them 1px. I used a batch action in Photoshop so no pain at all... and finally I had to change again the pixel in our side to make it look like before.